plugins/single_plugins/scale.cpp

```diff
@@ -529,7 +532,9 @@ class wayfire_scale : public wf::plugin_interface_t
                 auto vg = view->get_wm_geometry();
 
                 double scale_x = width / vg.width;
+                if(scale_x > max_scale_factor) scale_x = max_scale_factor;
```

Comment:
These conditionals need a space between the `if` and the first `(`, they need braces and the first brace should start on the line after the `if`. Also, both of these clauses should be placed directly above the `scale_x = scale_y = std::min(scale_x, scale_y);` line.

Reply:
changed the formatting and the logic slightly, so there is only one check